docs/src/docs/quick-start.md

```diff
@@ -1,13 +1,138 @@
 # KitOps Quick Start
 
 In this guide, we'll use ModelKits and the kit CLI to easily:
-* Package up a model, notebook, and datasets into a single ModelKit you can share through your existing mechanisms
+* Package up a model, notebook, and datasets into a single ModelKit you can share through your existing tools
 * Push the ModelKit package to a public or private registry
-* Grab just the things you need from the ModelKit for testing, integration, local running, or deployment
+* Grab only the assets you need from the ModelKit for testing, integration, local running, or deployment
 
 ## Before we start...
 
-Make sure you've got the [Kit CLI setup](./cli/installation.md) on your machine.
+1. Make sure you've got the [Kit CLI setup](./cli/installation.md) on your machine. If you'd like to learn more about ModelKits [check out the overview](./modelkit/intro.md). If you are wondering about specific flags you can check out everything in the [CLI command reference](./cli/cli-reference.md)
+
+1. Create and navigate to a new folder on your filesystem. We'd suggest calling it `KitStart` but any name works.
```

Comment:
I assume this is `2.` ?

```suggestion
2. Create and navigate to a new folder on your filesystem. We'd suggest calling it `KitStart` but any name works.
```

Reply:
With Markdown it should be smart enough to convert a series of 1. into sequential numbers, but let's do 2. to be safe.